toy_problem/src/main.cpp

```diff
@@ -376,10 +379,30 @@ class TaskPlanner
     while (!_priority_queue.empty())
     {
       auto top = _priority_queue.top();
+      auto display_queue = _priority_queue;
```

Comment:
I suggest merging the latest `master` into this branch and updating the `print_node()` function